src/picterra/client.py

```diff
@@ -943,6 +943,32 @@ def delete_vector_layer(self, vector_layer_id: UUID):
         if not resp.ok:
             raise APIError(resp.text)
 
+    def download_vector_layer_to_file(self, vector_layer_id: UUID, filename: str):
+        """
+        Downloads a vector layer
+
+        This a **beta** function, subject to change.
+
+        Args:
+            vector_layer_id: The id of the vector layer to remove
+            filename: existing file to save the vector layer in
+        """
+        if not (os.path.exists(filename) and os.path.isfile(filename)):
```

Comment:
It's also documented like this 2 lines above, but why does the file need to exist already? Couldn't it just be created if it doesn't exist yet?

Reply:
fixed